src/coreclr/debug/daccess/cdac.h

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef CDAC_H
+#define CDAC_H
+
+#include <cdac_reader.h>
+
+class CDAC final
+{
+public: // static
+    static CDAC* Create(uint64_t descriptorAddr, ICorDebugDataTarget *pDataTarget);
+
+public:
+    ~CDAC();
+
+    // This does not AddRef the returned interface
+    IUnknown* SosInterface();
+    int ReadFromTarget(uint64_t addr, uint8_t* dest, uint32_t count);
+
+private:
+    explicit CDAC(HMODULE module, uint64_t descriptorAddr, ICorDebugDataTarget* target);
```

Comment:
`explicit` here is fine, but unnecessary since it is private. The most common case for `explicit` when there is a constructor that takes a single type that is not a copy or move constructor. The reason for this is because of implicit conversions that can occur.